test/studies/comd/llnl/CoMD.chpl

```diff
@@ -838,7 +838,7 @@ tArray[timerEnum.TOTAL].stop();
       writef("%s : %12.4dr\n", t.name, t.duration);
   } else {
 
-    writeln("        Timer        # Calls    Avg/Call (s)   Total (s)    % Loop");
+    writeln("        stopwatch        # Calls    Avg/Call (s)   Total (s)    % Loop");
```

Comment:
Does the `good` file also need to change here?

Reply:
I changed it back to Timer, but this line never actually runs in our normal testing, so it didn't fail.